lib/run_loop/detect_aut/xcode.rb

```diff
@@ -39,7 +41,19 @@ def find_xcodeproj
       end
 
       # @!visibility private
-      def ignore_xcodeproj?(path)
+      def self.find_user_state_files
+        username = Etc.getpwuid.name
+        user_state_file_paths = []
+        Dir.glob("#{Dir.pwd}/**/xcuserdata/#{username}.xcuserdatad/UserInterfaceState.xcuserstate").each do |path|
+          next if ignore_xcodeproj?(path)
```

Comment:
`ignore_xcodeproj` is an instance method.  You cannot call it from a class method.